src/content/docs/browser/new-relic-browser/browser-apis/log.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: log
+type: apiDoc
+shortDescription: Capture a single log event.
+tags:
+  - Browser
+  - Browser monitoring
+  - Browser agent and SPA API
+metaDescription: Browser API call to capture data as a single log event.
+redirects:
+freshnessValidatedDate: never
+---
+
+## Syntax
+
+```js
+newrelic.log(message: string, options?: Object<{ customAttributes?: Object, level?: 'debug|error|info|trace|warn'}>)
+```
+
+Captures data as a single log event.
+
+## Requirements
+
+* Browser Pro, or Pro+SPA agent (v1.261.0 or higher)
+* If you're using npm to install the browser agent and using a non-standard implementation, you must enable the `logging` feature when instantiating the `BrowserAgent` class. For example, add the following in the`features` array:
+
+  ```js
+  import { Logging } from '@newrelic/browser-agent/features/logging'
+
+  const options = {
+    info: { ... },
+    loader_config: { ... },
+    init: { ... },
+    features: [
+      Logging
+    ]
+  }
+  ```
+
+  For more information, see the [npm browser installation documentation](https://www.npmjs.com/package/@newrelic/browser-agent#new-relic-browser-agent).
+
+## Description
+
+Upon executing this function with a valid message and elective options, the browser agent records the data as a single `log` event.  See the [Logs UI](https://docs.newrelic.com/docs/logs/ui-data/use-logs-ui/) for more information about log events. Any custom attributes supplied to the API call in the `options` argument (`options.customAttributes`) will be appended as top-level attributes on the log event. You can control the `logType` of the captured log by supplying a `level` to the `options` argument (`options.level`), which defaults to `info`.
+
+## Parameters
+
+<table>
+  <thead>
+    <tr>
+      <th width="25%">
+        Parameter
+      </th>
+
+      <th>
+        Description
+      </th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <tr>
+      <td>
+        `message`
+
+        _string_
+      </td>
+
+      <td>
+        Required. A string value which will be set to the `message` value of the created log event. The `message` property of the log event is the most visible property exposed on the log event and is used across the UI platform when displaying logs.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        `options`
+
+        _Object_
+      </td>
+
+      <td>
+        Optional. An object used for supplying optional configurations for the captured log. `options.customAttributes` is an object of key:val pairs that assigns a top-level property and value to the created log for each attribute supplied.  `options.level` is an enum that assigns a log level to the created log event.  The `level` must be one of: `debug | error | info | trace | warn`.  The log level defaults to `info` if not supplied.
```

Comment:
Could we rephrase this?

CURRENT: `options.level` is an enum that assigns a log level to the created log event.
PROPOSED:  The enum `options.level` assigns a log level to the created log event.